lms/services/d2l_api/_basic.py

```diff
@@ -3,9 +3,17 @@
 TOKEN_URL = "https://auth.brightspace.com/core/connect/token"
 """This is constant for all D2L instances"""
 
+API_VERSIONS = {
+    "le": "1.51",
```

Comment:
We are moving from 1.31 to 1.51 for this endpoint that has the documented side effect of needing a new scope for one of the endpoints we use.